R/directory_search.r

```diff
@@ -79,7 +79,13 @@ keyword_directory <- function(directory, keyword,
                               remove_equations = TRUE,
                               split_pattern = "\\p{WHITE_SPACE}{3,}",
                               full_names = TRUE, file_pattern = ".pdf",
-                              recursive = FALSE, max_search = NULL, ...) {
+                              recursive = FALSE, max_search = NULL,
+                              ocr_pdf_fun = function(...) {return("")},
+                              ocr_pdf_fun_params = nil,
+                              translate_fun = function(x,...) {return(x)},
+                              translate_fun_params = nil,
+                              translation_target_language="eng",

```

Comment:
Ultimately, I'd like these arguments to be passed as a list. This would allow fewer arguments to the primary function, but would require more thorough documentation. 

```suggestion
                              ocr_args = NULL,
                              translate_args = NULL,

```

Reply:
fixed in latest commit